### ISSUE TITLE
0006130:
add vacation templates

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#3119]
**Reported by pschuele on 23 Mar 2012 14:23**

add vacation templates
- allow define templates in Admin/Applications/Fmail (with placeholders)
- user can choose between the templates
- add email account signature
- show preview

**Comment by Gloirin** [comment]
Depends on #3184

**Comment by Gloirin** [comment]
Depends on #3185

**Comment by Gloirin** [comment]
Depends on #3186

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 9 May 2012 11:59**

branch: pu/6130_vacation -&gt; waiting to be merged into master

**Comment by Gloirin** [comment]
**Comment posted by pschuele on 10 Jul 2012 11:29**

Integrated in tine20:1,865  ( See http://ci.tine20.org/job/tine20/1865/ ).

Revision/Changeset: 3f7042035c404cc4bd1a270ae360b0af072bc69c 
Author: Philipp Schüle
Log: 
#6130: add vacation templates
Changed paths:
 tests/tine20/Felamimail/JsonTest.php
